contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,134 @@
+---
+title: What is a growth engineer (and how they help your startup)
+date: 2023-04-20
+author:
+    - ian-vanagas
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
```

Comment:
```suggestion
category: Startups
```

Since we mention startups in the headline, I think it's the better category for this one.